src/main/java/frc/robot/subsystems/Climb.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems;
+
+import frc.robot.Constants;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Climb extends SubsystemBase {
+    private WPI_TalonSRX climb_talon;
+    
+    public Climb() {
+        climb_talon = new WPI_TalonSRX(Constants.CLIMB_TALON);
+    }
+
+    public void set( ControlMode control_mode, double magnitude) {
```

Comment:
We should probably also have a "set" function which mirrors the default set from WPI_TalonSRX which has the ControlMode.PercentOutput.

@Override
	public void set(double speed) {
		_speed = speed;
		set(ControlMode.PercentOutput, _speed);
		feed();
	}